src/css/blocks/page.css

```diff
@@ -0,0 +1,91 @@
+.page {
+  position: relative;
+  z-index: 1;
+
+  padding-bottom: 10px;
+  padding-top: 10px;
+}
+
+.page__wrapper {
+  max-width: 50%;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.page::before {
+  content: " ";
+
+  border-bottom: 1px dashed rgba(255, 255, 255, 0.3);
+  border-top: 1px dashed rgba(0, 0, 0, 0.3);
+  display: block;
+  width: 100%;
+}
+.page::after {
```

Comment:
Can this be merged with `.page::before`declaration so that only `bottom: 0;` lives separately?